manifests/script-lab-edge.xml

```diff
@@ -17,6 +17,8 @@
     <AppDomain>https://github.com</AppDomain>
     <AppDomain>https://stackoverflow.com</AppDomain>
     <AppDomain>https://dev.office.com</AppDomain>
+    <AppDomain>https://localhost:3000</AppDomain>
```

Comment:
Sorry to be nitpicky.  But just to maintain consistency and order everywhere, could all four files -- which now share the same app domains, right? -- have the app-domain ordering for the script lab domains be consistent in the right order?

I.e., PROD => BETA => ALPHA => LOCALHOST.